packages/common/src/components/utils/Accordion.tsx

```diff
@@ -87,29 +112,43 @@ const Accordion: React.FC<Props> = ({
   const hasHeader = header !== null && header !== '' && header !== undefined
 
   const getHeader = () => {
-    if (!hasHeader) {
-      return (
-        <StyledSummary tabIndex={0} className="ic-expandable-button ic-inner ic-expandable-button--chevron iu-fs-400">
-          <Icon iconType={icon ? icon : ''} includeTooltip={includeIconTooltip} size={iconSize} />
-          <MandatoryIcon additionalStyles={mandatoryIconAdditionalStyles} display={displayMandatory as boolean} />
-          <h4 className={`${isCategory ? 'iu-fs-400' : 'iu-fs-300'} ${additionalStyles}`}>{title}</h4>
-        </StyledSummary>
-      )
-    } else {
+    if (isControl) {
       return (
         <StyledSummary tabIndex={0} className="ic-expandable-button ic-inner ic-expandable-button--chevron iu-fs-400">
           <Icon iconType={icon ? icon : ''} size={iconSize} />
           <MandatoryIcon display={displayMandatory as boolean} additionalStyles={mandatoryIconAdditionalStyles} />
-          <h5 className={`iu-fs-200 iu-lh-body ${additionalStyles}`}>{title}</h5>
+          <h5 className={`iu-fs-100 iu-fw-bold iu-lh-body close ${additionalStyles}`}>{title}</h5>
+          <h5 className={`iu-fs-100 iu-fw-bold iu-lh-body open ${additionalStyles}`}>{titleClose}</h5>
         </StyledSummary>
       )
+    } else {
+      if (hasHeader) {
+        return (
+          <StyledSummary tabIndex={0} className="ic-expandable-button ic-inner ic-expandable-button--chevron iu-fs-400">
+            <Icon iconType={icon ? icon : ''} size={iconSize} />
+            <MandatoryIcon display={displayMandatory as boolean} additionalStyles={mandatoryIconAdditionalStyles} />
+            <h5 className={`iu-fs-200 iu-lh-body ${additionalStyles}`}>{title}</h5>
+          </StyledSummary>
+        )
+      } else {
+        return (
+          <StyledSummary tabIndex={0} className="ic-expandable-button ic-inner ic-expandable-button--chevron iu-fs-400">
+            <Icon iconType={icon ? icon : ''} includeTooltip={includeIconTooltip} size={iconSize} />
+            <MandatoryIcon additionalStyles={mandatoryIconAdditionalStyles} display={displayMandatory as boolean} />
+            <h4 className={`${isCategory ? 'iu-fs-400' : 'iu-fs-300'} ${additionalStyles}`}>{title}</h4>
+          </StyledSummary>
+        )
+      }
     }
   }
```

Comment:
We have a lot of repetition in this function, instead of having large if-statements we can toggle individual parts. For example `StyledSummary` always looks the same.

Something like:
```typescript
const getTitle = () => {
  if (isControl) {
    return <h5 className={`iu-fs-100 iu-fw-bold iu-lh-body close ${additionalStyles}`}>{title}</h5>
  } else if (hasHeader) {
    return <h4 className={`${isCategory ? 'iu-fs-400' : 'iu-fs-300'} ${additionalStyles}`}>{title}</h4>
  }
  return <h5 className={`iu-fs-200 iu-lh-body ${additionalStyles}`}>{title}</h5>
}

const getHeader = () => {
  return (
    <StyledSummary tabIndex={0} className="ic-expandable-button ic-inner ic-expandable-button--chevron iu-fs-400">
      <Icon iconType={icon ? icon : ''} includeTooltip={!hasHeader && includeIconTooltip} size={iconSize} />
      {displayMandatory && <MandatoryIcon additionalStyles={mandatoryIconAdditionalStyles} display={true} />}
      {getTitle()}
      {titleClose && <h5 className={`iu-fs-100 iu-fw-bold iu-lh-body open ${additionalStyles}`}>{titleClose}</h5>}
    </StyledSummary>
  )
}
```

This is just a quick example, could probably be refined even more?